### ISSUE TITLE
Training loss curve

### ISSUE THREAD

**Issue by longnhatne** [viiika/Meissonic#25]
Hi, this is awesome!
Do you mind sharing your training curve?
I am trying to train Meissonic from scratch, I just want to see if my progress is aligning with yours.
Thank you very much.

**Comment by longnhatne** [comment]
Thank you for your support.
One more question, from your experience. What is the good value of cross-entropy loss to have a acceptable or really good output images?

**Comment by viiika** [comment]
The loss value alone is not meaningful. A more diverse training dataset often results in a higher loss value. For example, on some public training datasets, the optimal loss might be around 6.7 or 6.3, whereas on simpler datasets, it could be as low as 1 or 2. Additionally, training for a longer period does not necessarily indicate better performance; it may lead to overfitting on your dataset, even if you observe a slight decrease in the loss value.

Therefore, it’s more important to focus on the actual image quality during inference at specific training steps.

**Comment by longnhatne** [comment]
Oh, that is an exciting information. Good to know it.
Thank your very much
